apps/api/src/queries/buildViewModelForResource/viewModels/vocabulary-list.view-model.ts

```diff
@@ -49,8 +57,68 @@ export class VocabularyListEntryViewModel extends BaseDomainModel {
 
         this.variableValues = { ...variableValues };
     }
+
+    public canUserWithGroups(userWithGroups: CoscradUserWithGroups) {
```

Comment:
I suppose the intention is for this to be a generic helper method on all resource view models.